src/modules/sync/diff.ts

```diff
@@ -0,0 +1,13 @@
+import { ManagementClient } from "@kontent-ai/management-sdk";
+
+type DiffConfig = DiffOptions;
```

Comment:
I don't understand why this type exists. This is the only usage of the `DiffOptions` type, you can instead rename it to `DiffConfig` and remove this type if you don't like the `DiffOptions` name. 😄

Reply:
at first it combied two types together :D for some reason i did not remove it when deleted the other type :/